std_msgs/Header header

float64 scale_by
geometry_msgs/PoseStamped live_hand_pose
geometry_msgs/Pose[] cartesian_wayposes

bool in_natural_control_mode
bool orientation_free
bool position_unlimited
bool position_forward_only

bool in_navigation_mode

geometry_msgs/PoseStamped end_effector_pose

